### ISSUE TITLE
Flutter support

### ISSUE THREAD

**Issue by wxlpp** [NanoMichael/cLaTeXMath#32]
flutter is a great cross-platform framework. If you support flutter, it will make cross-platform extremely simple.

**Comment by NanoMichael** [comment]
Yes, I'm planning to do this. But the topmost thing I'm doing now is the transfer from plain ASCII TeX to Unicode-math supported modern LaTeX. Anyway, the Flutter version is in the plan.

**Comment by thaminmg** [comment]
May I know the status for flutter support?

**Comment by NanoMichael** [comment]
@thaminmg You may want to take a look at this [link](https://github.com/NanoMichael/MicroTeX/tree/flutter-support/platform/flutter/microtex).

But you may need to patch this [feature](https://github.com/NanoMichael/engine/pull/1) and re-compile your flutter engine to make it work.
